<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReminderDialog</class>
 <widget class="QDialog" name="ReminderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        font-family: &quot;Microsoft YaHei&quot;;
        font-size: 14px;
        color: #333333;
        background-color: #f7f9fb;
    }

    QPushButton {
        background-color: #4CAF50;
        color: white;
        padding: 6px 12px;
        border: none;
        border-radius: 4px;
    }

    QPushButton:hover {
        background-color: #45a049;
    }

    QLineEdit {
        border: 1px solid #ccc;
        padding: 4px;
        border-radius: 4px;
        background-color: #ffffff;
    }

    QLabel {
        color: #2c3e50;
    }

    QTableWidget {
        background-color: white;
        gridline-color: #dcdcdc;
        selection-background-color: #cceeff;
    }

    QHeaderView::section {
        background-color: #e0e0e0;
        padding: 4px;
        border: 1px solid #dcdcdc;
    }</string>
  </property>
  <widget class="QListWidget" name="listReminders">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>217</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnConfirm">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnAddReminder">
      <property name="text">
       <string>添加提醒</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnRemoveReminder">
      <property name="text">
       <string>删除提醒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置提醒</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkRemindStart">
      <property name="text">
       <string>开始提醒</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkRemindEnd">
      <property name="text">
       <string>结束提醒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>119</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSpinBox" name="spinMinutesBefore">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>分钟</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
